lepidopter-fh/configure.sh

```diff
@@ -3,3 +3,10 @@ set -ex
 
 # Set locatime to UTC
 cp /usr/share/zoneinfo/UTC /etc/localtime
+
+# Checkout lepidopter-update repo
+git clone https://github.com/OpenObservatory/lepidopter-update \
+    /opt/ooni/lepidopter-update
+
+# Enable lepidopter-update systemd service
+systemctl enable lepidopter-update
```

Comment:
Is there any specific need for the lepidopter-updater service to be running inside the image build?